src/StackExchange.Redis/PhysicalConnection.cs

```diff
@@ -649,7 +649,8 @@ internal void OnBridgeHeartbeat()
                     var timeout = bridge.Multiplexer.AsyncTimeoutMilliseconds;
                     foreach (var msg in _writtenAwaitingResponse)
                     {
-                        if (msg.HasAsyncTimedOut(now, timeout, out var elapsed))
+                        // We only handle async timeouts here, synchronous timeouts are handled upstream.
```

Comment:
The direction of 'upstream' isn't that conventionally clear to me. Does it mean before OnBridgeHeartbeat would be called (since upstream is the 'source' of the data flowing)? Or does it just mean 'somewhere else in the class which normally calls into this?'

Reply:
Adding an additional comment here to clarify:
```cs
// Those sync timeouts happen in ConnectionMultiplexer.ExecuteSyncImpl() via Monitor.Wait.
```